### PR TITLE
(dbt) Add iris_id dimension to enriched and mart models

### PR DESCRIPTION
**Pull request by MatthieuRipollPassCulture**
# DE PR

## Describe your changes

The purpose of this PR is to add the `iris_id` dimension to the following dimensions :
- `user`
- `venue`
- `institution`

We are therefore adding the iris_id dimension to the models `int_applicative__user`, `int_applicative__venue`, `int_applicative__institution` (new models), `venue`, and `institution_location` (existing models). We prefix the column name each time to specify whether it is the iris_id of the user, venue, or institution (`user_iris_internal_id`, `venue_iris_internal_id`, etc.).

We then retrieve this dimension in the enriched models:
- `enriched_user_data`
- `enriched_collective_offer`
- `enriched_collective_booking`
- `mrt_global__venue`
- `mrt_global__user`
- `mrt_global__offer`
- `mrt_global__booking`

We also add the dimension to models that are intermediaries to make it available everywhere, such as `mrt_global__stock` and `mrt_global__venue_unverified`.

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/analytics/institution_locations.sql
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/analytics/institution_locations.sql:1]: Penser à ajouter le champs iris dans enriched_institution_data

File: orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__offer.sql
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__offer.sql:1]: Il faut aussi ajouter le champ dans enriched_offer_data qui lui est en double run

Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__offer.sql:1]: Ici, ajouter les champs de mrt_global__stock directment et les récupérer

File: orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__venue.sql [hunk 1]
```diff
@@ -68,6 +68,7 @@ SELECT venue_id,
     offerer_is_active,
     venue_pc_pro_link,
     partner_id,
+    venue_iris_internal_id
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/mart/global/mrt_global__venue.sql:71]: ici l'ajouter aussi dans mrt_global__venue_unverified

**Review by carolinemestre-passculture — COMMENTED** [review]
Self review

**Review by cdelabre — DISMISSED** [review]
Top, merci ⭐ .

**Review by cdelabre — DISMISSED** [review]

**Review by carolinemestre-passculture — APPROVED** [review]

**Pull request merged**